src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/Microsoft.CodeAnalysis.Analyzer.Testing.csproj

```diff
@@ -13,12 +13,42 @@
     <PackageTags>Roslyn Analyzer Test Framework Common</PackageTags>
   </PropertyGroup>
 
+  <Choose>
+    <When Condition="'$(TargetFramework)' == 'net452'">
+      <PropertyGroup>
+        <NuGetApiVersion>4.5.3</NuGetApiVersion>
+      </PropertyGroup>
+    </When>
+    <When Condition="'$(TargetFramework)' == 'net46'">
+      <PropertyGroup>
+        <NuGetApiVersion>4.9.4</NuGetApiVersion>
+      </PropertyGroup>
+    </When>
+    <When Condition="'$(TargetFramework)' == 'netstandard1.5'">
+      <PropertyGroup>
+        <NuGetApiVersion>4.6.4</NuGetApiVersion>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <NuGetApiVersion>5.3.1</NuGetApiVersion>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
```

Comment:
What is this about?

Reply:
TFM support in these APIs is a moving target. These are the newest versions to support each of the possible testing scenarios.